packages/perspective-viewer/src/less/material.less

```diff
@@ -90,62 +63,67 @@ perspective-viewer {
         &:before {
             font-family: "Material Icons";
             content: "more_vert" !important;
+=======
+        #columns_container #inactive_columns perspective-row .row_draggable {
+            height: 26px;
+>>>>>>> Stashed changes
         }
     }
 
-    #app {
+    &.dragging #app {
 
         #side_panel {
 
             #columns_container {
 
-                #active_columns {
-
-                    perspective-row:before {
-                        margin-left: 48px;
-                        left: 48px;
+                #inactive_columns {
+                    perspective-row:hover .is_visible {
+                        opacity: 0;
                     }
+                }
 
-                    perspective-row {
-                        .is_visible:before {
-                            font-family: "material icons";
-                            margin-top: 1px;
-                            margin-left: -8px;
-                            font-size: 16px;
-                            content: "\E834";
-                        }
-                    }
+                #active_columns perspective-row[drop-target] .row_draggable, .rrow ul {
+                    margin-top: 0;
                 }
             }
+        }
 
+        #top_panel .rrow > * {
+            border-top: 0 !important;
+            border-left: 0 !important;
+            border-right: 0 !important;
+            margin-top: 0;
+            margin-left: 0;
         }
     }
 
-    #app #top_panel {
-
-        #row_pivots label:before {
-            content: "Group By";
+    #config_button {
+        margin-top: 14px;
+        margin-left: 7px;
+        &:before {
+            font-family: "Material Icons";
+            content: "more_vert" !important;
         }
+    }
 
-        #column_pivots label:before {
-            content: "Split By";
-        }
+    .psp-text-field__input + label {
+        padding-left: 0;
     }
 }
 
-perspective-viewer #app.columns_horizontal {
+perspective-viewer #app {
 
-    #side_panel {
-        max-width: 500px !important;
-        padding: 24px 5px 0 0;
+    &.columns_horizontal {
 
-        #divider {
-            display: none;
+        #side_panel {
+            max-width: 500px !important;
+            padding: 24px 5px 0 0;
+
+            #divider {
+                display: none;
+            }
         }
```

Comment:
These seem to revert a [commit I made on responsive-sidebar](https://github.com/jpmorganchase/perspective/commit/655dd7bbd2cb40f55c3b59dd87308f8b4c2860da) before merging, to consolidate `.columns_horizontal`.

Reply:
Nevermind - I misread, this works as intended.

Reply:
I didnt know you could do this...

Reply:
Changes fixed.